### ISSUE TITLE
`require-description-complete-sentence` rule provides TypeError without empty JSDoc

### ISSUE THREAD

**Issue by krutoo** [gajus/eslint-plugin-jsdoc#290]
Hello, rule `require-description-complete-sentence` fails ESLint 

![image](https://user-images.githubusercontent.com/13140130/59997131-9ebc6e80-9675-11e9-90e2-18b77c310ffd.png)

**Comment by gajus** [comment]
What is the actual comment that this rule is failing with?

**Comment by krutoo** [comment]
> What is the actual comment that this rule is failing with?

I write this code:
```javascript
import React from 'react';

/**
 *
 */
const Product = () => (
  <div>
    Test
  </div>
);

export default Product;
```

**Comment by brettz9** [comment]
Yes, this is now fixed in 8.5.1, thanks!

**Comment by brettz9** [comment]
While the error has been fixed, note that this only validates when there is a description. If you want to require that there is a description at all, you need `require-description`.

**Comment by krutoo** [comment]
Yes, this issue was resolved and can be closed. Thank you for fastest release!
